napari/benchmarks/benchmark_labels_layer.py

```diff
@@ -3,6 +3,7 @@
 # or the napari documentation on benchmarking
 # https://github.com/napari/napari/blob/master/docs/BENCHMARKS.md
 import numpy as np
+from skimage.draw import disk
```

Comment:
Sorry i missed this on the last review @DragaDoncila but can we avoid using skimage here and just manually construct some coordinates. `skimage` isn't a napari dependency right now and someone could try and run the benchmarks in an environment without it. We might make it one in the future, but till then best not to use if possible